ui-tests/tests/replacePerItem.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { test, galata } from '@jupyterlab/galata';
```

Comment:
locator using id with index are not robust. The chance a code modification will break the test (without breaking the feat' is high).